masterlist.yaml

```
@@ -8174,11 +8233,29 @@ plugins:
       - 'PerkusMaximus_Warrior.esp'
       - 'PerkusMaximus_CombatPlus.esp'
       - 'SkyRe_Main.esp'
+    msg:
+      - <<: *compatNotes
+        subs: [ 'https://requiem.atlassian.net/wiki/spaces/RSSE/pages/2658926593/Compatibility+Advice' ]
+      - *includesMBBF
+  - name: 'Requiem - No Overencumbered Messages.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/60888/'
+        name: 'Requiem - The Roleplaying Overhaul'
+    msg:
+      - <<: *alreadyInX
+        condition: 'active("Requiem - No Messages.esp")'
+        subs: [ 'Requiem - No Messages.esp' ]
```

Sorry, I missed this too. The `condition` key should be sorted after the `subs` key:

```suggestion
        subs: [ 'Requiem - No Messages.esp' ]
        condition: 'active("Requiem - No Messages.esp")'
```